spec.html

```diff
@@ -45444,8 +45444,8 @@ <h1>IfAbruptRejectPromise ( _value_, _capability_ )</h1>
 
       <emu-clause id="sec-promisereaction-records">
         <h1>PromiseReaction Records</h1>
-        <p>The PromiseReaction is a Record value used to store information about how a promise should react when it becomes resolved or rejected with a given value. PromiseReaction records are created by the PerformPromiseThen abstract operation, and are used by the Abstract Closure returned by NewPromiseReactionJob.</p>
-        <p>PromiseReaction records have the fields listed in <emu-xref href="#table-promisereaction-record-fields"></emu-xref>.</p>
+        <p>The <dfn variants="PromiseReaction Records">PromiseReaction Record</dfn> is a Record value used to store information about how a promise should react when it becomes resolved or rejected with a given value. PromiseReaction Records are created by the PerformPromiseThen abstract operation, and are used by the Abstract Closure returned by NewPromiseReactionJob.</p>
```

Comment:
```suggestion
        <p>A <dfn variants="PromiseReaction Records">PromiseReaction Record</dfn> is a Record value used to store information about how a promise should react when it becomes resolved or rejected with a given value. PromiseReaction Records are created by the PerformPromiseThen abstract operation, and are used by the Abstract Closure returned by NewPromiseReactionJob.</p>
```
while we're here, for consistency with the other kinds of records we define.